### PR TITLE
docs: reverted the doc menu categories

### LINKED ISSUE #2073: Docs: Remove menu categories
This is a follow-up after:
 - https://github.com/FuelLabs/fuels-ts/issues/1899

The idea is to revert the addition of main categories, like `Essentials`, `Basics`, etc.

This is to make it look more fluid, like the [other docs](https://rust.fuel.network/v0.57.0/) we have that uses [mdbook](https://github.com/rust-lang/mdBook).

### PR DESCRIPTION
**Pull request by petertonysmith94**
Closes #2073 

---

- Reverted changes to the menu categories.

### CODE REVIEW

File: apps/docs/.vitepress/config.ts [hunk 1]
```diff
@@ -57,8 +57,8 @@ export default defineConfig({
         ],
       },
       {
-        text: 'Basics',
-        link: '/guide/basics/',
+        text: 'Guide',
```
Comment (arboleya) [inline apps/docs/.vitepress/config.ts:60]: To be as similar as possible to the docs hub, perhaps we should remove all sections, including `Introduction` and `Guide`, and have a flat list.

Comment (arboleya) [reply apps/docs/.vitepress/config.ts:60]: Could all items look like the `Getting Started` and `Creating a Fuel dApp` items?

**Review by arboleya — DISMISSED** [review]

File: apps/docs/.vitepress/config.ts (hunk 1 above)
Comment (arboleya) [reply apps/docs/.vitepress/config.ts]: Unrelated to this PR, just cross-referencing these here:
 - https://github.com/FuelLabs/fuels-ts/issues/2086
 - https://github.com/FuelLabs/fuels-ts/issues/2087

**Review by maschad — CHANGES_REQUESTED** [review]

**Review by maschad — APPROVED** [review]
lgtm!

**Review by arboleya — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

**Pull request merged**